src/components/modals/StepConnectDevice.js

```diff
@@ -2,28 +2,29 @@
 
 import React from 'react'
 
-import type { Currency } from '@ledgerhq/currencies/lib/types'
-
+import type { Account, Currency } from '@ledgerhq/currencies/lib/types'
 import type { Device } from 'types/common'
 
 import DeviceConnect from 'components/DeviceConnect'
 import DeviceMonit from 'components/DeviceMonitNew'
 
 type Props = {
-  currency: Currency | null,
+  account?: Account,
+  currency?: Currency | null,
```

Comment:
the `T | null` pattern in Flow, should be replaced by `?T`

to be checked but I also think `currency?: Currency` is enough, no need to do `currency?: ?Currency`